README.md

```diff
@@ -2,41 +2,64 @@
 
 [![Build Status](https://dev.azure.com/sitecorejss/sitecore-jss-ci/_apis/build/status/Sitecore.jss?branchName=master)](https://dev.azure.com/sitecorejss/sitecore-jss-ci/_build/latest?definitionId=1?branchName=master) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](contributing.md#pull-requests)
 
-This repository contains source code for all Sitecore JSS packages as well as samples to help you get started using Sitecore JSS.
+This repository contains source code for all Sitecore JSS packages and sample applications to help you get started using Sitecore JavaScript Services (JSS).
```

Comment:
More of an FYI than a change request:

Technically, the name was officially changed from JavaScript Services to JavaScript SDK after the .NET Core SDK was release. But the old name is still used all over the place.